### PR TITLE
General formatting

### LINKED ISSUE #42: General reformatting
Code could use a more standard way of arranging imports, spacing and other general stuff. I'll propose something in a PR.

### PR DESCRIPTION
**Pull request by outime**
It doesn't change any features or fixes any bugs so it should be an easy review. I recommend to take a look at the [PEP 8](https://www.python.org/dev/peps/pep-0008/) or at least follow some consistent code guidelines.

Fixes #42

### CODE REVIEW

File: lib/gmaps.py [hunk 1]
```diff
@@ -188,7 +185,7 @@ def get_confidence(data, cfg):
             if all([loc in locations[hash]["locations"] for loc in locations[hash2]["locations"]]):
                 to_del.append(hash2)
     for hash in to_del:
-        del(locations[hash])
+        del (locations[hash])
```
Comment (mxrch) [inline lib/gmaps.py:188]: Nice, I always thought it was a function, I learn something

Comment (outime) [reply lib/gmaps.py]: I just removed the brackets as well :)

File: config.py [hunk 2]
```diff
@@ -1,13 +1,14 @@
 cfg = dict(
-    regexs = {
-"albums": r'href=\"\.\/albumarchive\/\d*?\/album\/(.*?)\" jsaction.*?>(?:<.*?>){5}(.*?)<\/div><.*?>(\d*?) ',
-"photos": r'\],\"(https:\/\/lh\d\.googleusercontent\.com\/.*?)\",\[\"\d{21}\"(?:.*?,){16}\"(.*?)\"',
-"review_loc_by_id": r'{}\",.*?\[\[null,null,(.*?),(.*?)\]'
-},
+    regexs={
```
Comment (mxrch) [inline config.py:2]: I think it reduces readability, are you sure it is in the pep-8 convention ?
It's seems weird, I'd rather not

Comment (outime) [reply config.py:2]: The dictionary is described in multiple lines, hence indentation facilitates reading. It's a quite standard way and it's much easier to see that `cfg` has all those kv.. Or are you talking about something else?

Comment (mxrch) [reply config.py:2]: @outime Yes the indentations are okay, I was talking about the "regexs={", isn't it better with whitespaces ?

Comment (outime) [reply config.py:2]: @mxrch ah gotcha! Let me show you an alternative way in a second.

Comment (outime) [reply config.py]: @mxrch here's one alternative https://github.com/mxrch/GHunt/pull/43/commits/cc2aa8c46ee033d56dfda8015cc9b185c0362689 if you don't like it then I can drop it and go back to the previous one with spaces.

**Comment by mxrch** [comment]
Sorry but I rather keep the dict() method in the config !

**Comment by outime** [comment]
@mxrch changed back, re-check and merge if ok.

**Comment by mxrch** [comment]
Thank you !

**Pull request merged**